vcpkg/commands/edit.md

```diff
@@ -0,0 +1,56 @@
+---
+title: vcpkg edit
+description: Command-line reference for the vcpkg edit command. Opens a port's folder in a new code editor window.
+author: vicroms
+ms.author: viromer
+ms.date: 7/24/2023
+ms.prod: vcpkg
+---
+# vcpkg edit
+
+## Synopsis
+
+```Console
+vcpkg edit <ports> [--options]
+```
+
+## Description
+
+Opens a port's corresponding `ports` folder in a new _Visual Studio Code_ window.
```

Comment:
I'm not sure we should introduce the term "`ports` folder" here; I don't believe we use that language elsewhere. I'd instead suggest something like "definition" or avoid it entirely -- "Opens a port for editing in ...".